### DISCUSSION TITLE
Disabling automatic loaders?

### DISCUSSION THREAD

**Discussion by ZSPFX** [mercurius-js/mercurius#937] [Q&A]
Hi! I'm trying to migrate my graphql code from Apollo to Mercurius and ran into some issues related to "automatic loader" that will not allow the resolver field for the query that'll cause n + 1 to be executed. And the only way to solve that is to have a loader for that field.

I tried making the loader for it but the way resolver was previously structured while working with Apollo made making loaders way difficult than it should've been.

I wonder is there any way or config to temporary disable this automatic loader behaviour? Or would it be possible to add one to the later updates.

No worries if it's not possible, thanks!

**Comment by mcollina** [comment]
I don't understand the question. You need to opt-in to use loaders, otherwise you just use plain resolvers.

**Reply by ZSPFX** [reply]
I tried making a reproducible sample repo for you but it works just fine there?? Which is extremely odd.
Here's the recording I've made from an actual production repo that has the of issue I was mentioning about [https://streamable.com/v1aki6](https://streamable.com/v1aki6)

do you have any idea of what could possibly be causing this?
